ts/src/huobi.ts

```diff
@@ -2928,11 +2928,11 @@ export default class huobi extends Exchange {
                 maxWithdraw = this.safeNumber (chainEntry, 'maxWithdrawAmt');
                 const withdrawStatus = this.safeString (chainEntry, 'withdrawStatus');
                 const depositStatus = this.safeString (chainEntry, 'depositStatus');
-                const withdrawEnabled = (withdrawStatus === 'allowed');
-                const depositEnabled = (depositStatus === 'allowed');
-                withdraw = (withdrawEnabled) ? withdrawEnabled : withdraw;
-                deposit = (depositEnabled) ? depositEnabled : deposit;
-                const active = withdrawEnabled && depositEnabled;
+                const isWithdrawalEnabled = (withdrawStatus === 'allowed');
+                const isDepositEnabled = (depositStatus === 'allowed');
+                isTokenDepositable = isDepositEnabled || isTokenDepositable;
+                isTokenWithdrawable = isWithdrawalEnabled || isTokenWithdrawable;
+                const active = isWithdrawalEnabled && isDepositEnabled;
```

Comment:
Sometime it's inside a variable, sometime it's directly inside the currency object at the end, should we choose between one and stick to it?